src/api/dataSource/NodeDataSource.ts

```diff
@@ -31,6 +31,14 @@ export interface Application {
   version: string;
 }
 
+export interface BlobApplication {
```

Comment:
What is difference between Application and BlobApplication?

Reply:
BlobApplication is the updated object from admin api and I still want to leave Application available as we might need to update it again

Reply:
<img width="525" alt="Screenshot 2024-08-08 at 11 59 11" src="https://github.com/user-attachments/assets/19434a4d-bf7a-4895-9f21-332ea74cdf91">

Reply:
But can we use Application for both?

Reply:
```
BlobApplication {
  id: string;
  blob: string;
  version: string;
  source: string;
  contract_app_id: string;
}
```

```
interface Package {
  id: string;
  name: string;
  description: string;
  repository: string;
  owner: string;
}

interface Application extends Package {
  version: string;
  contract_app_id: string;
}
```
I would leave it as is

Reply:
Updated